source/elements/oneDPL/source/parallel_api/parallel_range_api.rst

```diff
@@ -0,0 +1,321 @@
+.. SPDX-FileCopyrightText: Contributors to the oneAPI Specification project.
+..
+.. SPDX-License-Identifier: CC-BY-4.0
+
+Parallel Ranges API
+-------------------
+
+oneDPL provides variations of algorithms that work with ranges defined in the `C++ Standard`_, 6th edition (C++20).
+These algorithms execute according to a oneDPL execution policy supplied as the first argument, similarly to other
+oneDPL algorithms.
+
+The oneDPL parallel range algorithms rely on the functionality of C++20 and are not available in the code
+compiled for earlier editions of the standard.
+
+The parallel range algorithms reside in ``namespace oneapi::dpl::ranges``. Same as the range algorithm functions
+defined by the `C++ standard` in ``namespace std::ranges``, they cannot be found by argument-dependent name lookup.
```

Comment:
Do we want just require (on the specification level) that all of them are global callable objects (independently of where C++ standard goes)? With that requirement we give extra benefits for the users.

Reply:
Which specifically benefits would that give? Not theoretical but real ones, something that you can imagine being useful in practice.

Reply:
passing algorithms as arguments to a function. This is the only practical difference I can see. It's guaranteed to work. With having "special function that..." it's not guaranteed to work. That's it

Reply:
Yes, that's a theoretical possibility. But I do not know why one would use it in practice, given that calling signatures of those algorithms are quite different and no clear reason why someone would want anything like that.
In simple words, I see practical value of this being (nearly) absent.

Reply:
I have added a note about function object based implementations.